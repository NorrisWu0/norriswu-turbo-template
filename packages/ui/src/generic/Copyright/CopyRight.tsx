import Image from "next/image";

import vercel from "./assets/vercel.svg";

export function CopyRight() {
  return (
    <div className="">
      <a
        href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        target="_blank"
        rel="noopener noreferrer"
        className="ui-flex ui-items-center ui-justify-center ui-gap-2 sm:ui-p-4"
      >
        {"By "}
        <Image
          src={vercel}
          alt="Vercel Logo"
          width={100}
          height={24}
          className="ui-invert"
        />
      </a>
    </div>
  );
}
