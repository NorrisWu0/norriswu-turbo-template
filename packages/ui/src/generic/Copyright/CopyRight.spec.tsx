import { render } from "@testing-library//react";

import { CopyRight } from "./CopyRight";

describe("CopyRight", () => {
  it("should match snapshot", () => {
    const { container } = render(<CopyRight />);
    expect(container).toMatchSnapshot();
  });
});
