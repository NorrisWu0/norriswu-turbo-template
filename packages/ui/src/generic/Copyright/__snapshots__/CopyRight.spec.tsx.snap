// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CopyRight should match snapshot 1`] = `
<div>
  <div>
    <a
      href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
      rel="noopener noreferrer"
      target="_blank"
    >
      By 
      <img
        alt="Vercel Logo"
        class="chakra-image css-gdnsru"
      />
    </a>
  </div>
</div>
`;
